projects/plugins/boost/app/assets/src/js/pages/settings/elements/RatingCard.svelte

```diff
@@ -0,0 +1,41 @@
+<!--
+	This component shows a prompt to rate Boost if scores improved after enabling a feature.
+-->
+<script>
+	import { __, sprintf } from '@wordpress/i18n';
+	import CloseButton from '../../../elements/CloseButton.svelte';
+	import { createEventDispatcher } from 'svelte';
+	import slideRightTransition from '../../../utils/slide-right-transition';
+	import { makeAdminAjaxRequest } from '../../../utils/make-admin-ajax-request';
+
+	export let improvement;
+
+	const dispatch = createEventDispatcher();
+	function disableRatingPrompt() {
+		makeAdminAjaxRequest( { action: 'set_show_rating_prompt', value: false } );
```

Comment:
The dispatch is for closing the rating prompt. I have pushed a change with async-await. However, I have some feeling that it might be ok to dismiss the prompt. I mean even if we fail to disable it permanently, it might be good to at least not continue showing the prompt for the moment.